cachito/workers/pkg_manager.py

```diff
@@ -70,6 +79,11 @@ def resolve_gomod_deps(archive_path, request_id):
             if len(parts) == 1:
                 # This is the application itself, not a dependency
                 continue
+            elif len(parts) == 3:
```

Comment:
What are your thoughts on converting this piece into a switch statement instead? I think it would look cleaner.

Reply:
@yashvardhannanavati Python doesn't support switch statements.

Reply:
Ah, that's right. My bad. Java is playing with my mind again.